README.md

```diff
@@ -189,7 +189,7 @@ If you like this project reach me out.
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
+> Give credit to  Nelson Sakwa the author of the original design.
 
 I would like to thank Microverse for giving me this opportunity to learn and everyone whose code was used.
 
```

Comment:
- [ ] It's great that you have Acknowledged Nelson Sakwa for his design but it would be nice if you make his name a link to his account so viewers of your application will easily access the author's account for credibility purposes as suggested by the previous reviewer :blush: It would look like below :arrow_down:
```
I would like to give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
```